code/src/components/NotDoneTasks.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import tasks from "reducers/tasks";
+import AddTask from "./AddTask";
+import state from '../assets/state.png'
+import trash from '../assets/trash.svg'
+
+const NotDoneTasks = () => {
+
+    const items = useSelector((backpack) => backpack.tasks.items)
+
+    const notDoneTasks = items.filter((taskItem) => !taskItem.isCaught);
+
+    const dispatch = useDispatch()
+
+    const onTaskToggle = (taskId) => {
+        dispatch(tasks.actions.toggleItem(taskId))
+    }
+
+    const onDeleteTask = (taskId) => {
+        dispatch(tasks.actions.deleteTask(taskId))
+    }
+
+    const onClickAllDone = () => {
+        dispatch(tasks.actions.allTasksDone())
+    }
+
+    const notDoneSummary = notDoneTasks.length
+
+     if (notDoneTasks < 1) {
+         return (
+            <>
+            <section className="section-wrapper">
+            <img className="tasklist-img" src={state} alt="tasklist" />
+            <AddTask />
```

Comment:
Perhaps you want to create a component for Empty State, so we do not need to rewrite these line of code many times